src/core/net/srp_client.cpp

```diff
@@ -1137,7 +1138,7 @@ Error Client::AppendUpdateLeaseOptRecord(Message &aMessage) const
     }
     else
     {
-        leaseOption.SetLeaseInterval(mLeaseInterval);
+        leaseOption.SetLeaseInterval(GetTtl());
```

Comment:
Curious why this change here. Should this not remain based on lease time?

Reply:
I agree. I think we may want to stay `mLeaseInterval`.